nemo_skills/pipeline/eval.py

```
@@ -424,51 +467,220 @@ def eval(
+
+        if jobs:
+            pipeline = Pipeline(
+                name=expname,
+                cluster_config=cluster_config,
+                jobs=jobs,
+                reuse_code=reuse_code,
+                reuse_code_exp=reuse_code_exp,
+                skip_hf_home_check=skip_hf_home_check,
+            )
+            handles = pipeline.run(dry_run=dry_run, _reuse_exp=exp, sequential=sequential)
+            job_name_to_handle = dict(zip(job_names, handles))
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)